cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,95 @@
+            return https_fn.Response(
+                status=400,
+                response=f"Bad Request - Unsupported data source: {data_source}"
+            )
+
+    except auth.InvalidIdTokenError:
+        return https_fn.Response(status=401, response="Unauthorized - Invalid token")
+    except auth.ExpiredIdTokenError:
+        return https_fn.Response(status=401, response="Unauthorized - Expired token")
+    except auth.RevokedIdTokenError:
+        return https_fn.Response(status=401, response="Unauthorized - Revoked token")
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)